test/cmdlineTests.sh

```diff
@@ -64,18 +70,28 @@ fi
 
 function ask_expectation_update()
 {
-    if [ $INTERACTIVE ]
+    if [[ $INTERACTIVE != "" ]]
     then
         local newExpectation="${1}"
         local expectationFile="${2}"
-        while true;
-        do
-            read -p "(u)pdate expectation/(q)uit? "
-            case $REPLY in
-                u* ) echo "$newExpectation" > $expectationFile ; break;;
-                q* ) exit 1;;
-            esac
-        done
+
+        if [[ $AUTOUPDATE == true ]]
+        then
+            echo "$newExpectation" > "$expectationFile"
+            printLog "File $expectationFile updated to match the expectation."
```

Comment:
Just wondering if the two lines above could be put inside a function that is called here and inside the case statement below

Reply:
They could, though that does not save us that much because the function is longer than the code it replaces :)

Still, might better for maintainability so I added it anyway.